go/install/install_darwin.go

```diff
@@ -818,6 +841,38 @@ func UninstallKBFS(context Context, mountDir string, forceUnmount bool, log Log)
 	if !empty {
 		return fmt.Errorf("Mount has files after unmounting: %s", mountDir)
 	}
+	return nil
+}
+
+// UninstallKBFS uninstalls all KBFS services, unmounts and optionally removes the mount directory
+func UninstallKBFS(context Context, mountDir string, forceUnmount bool, log Log) error {
+	err := UninstallKBFSServices(context, log)
+	if err != nil {
+		log.Warning("Couldn't stop KBFS: %+v", err)
+		// Continue despite the error, since the uninstall doesn't
+		// seem to be resilient against the "fallback" PID getting out
+		// of sync with the true KBFS PID.  TODO: fix the fallback PID
+		// logic?
+	}
+
+	err = unmount(mountDir, forceUnmount, log)
+	if err != nil {
+		return err
+	}
+	// For older systems, check `/keybase` too, just in case.
+	var oldMountDir string
+	switch context.GetRunMode() {
+	case libkb.ProductionRunMode:
+		oldMountDir = "/keybase"
+	case libkb.StagingRunMode:
+		oldMountDir = "/keybase.staging"
+	default:
+		oldMountDir = "/keybase.devel"
+	}
+	err = unmount(oldMountDir, forceUnmount, log)
+	if err != nil {
+		log.Debug("Error unmounting old mount dir %s: %v", oldMountDir, err)
```

Comment:
Why aren't we returning the error here? It's nonfatal in the caller anyway...

Reply:
`UninstallKBFSOnStop()` doesn't ignore the error, and I don't think it's worth risking breaking things just for the old `/keybase` link.